gatsby-node.js

```diff
@@ -78,15 +78,19 @@ exports.onCreateNode = async ({
 }) => {
   const { createNodeField, createNode } = actions
   if (node.internal.type === `apps`) {
-    return createRemoteFileNode({
-      url: node.imageUrl,
+    let remoteNode = {
+      url: node.imageUrl.trim() ? node.imageUrl.trim() 'https://via.placeholder.com/150',
       parentNodeId: node.id,
       store,
       cache,
       createNode,
       createNodeId,
       auth: _auth,
-    })
+    };
+    if( !node.imageUrl.trim()) ) remoteNode['name'] = 'lastCommit'
```

Comment:
No, that is not what I meant. But I will fix that. After resolving the promise for `createRemoteFileNode` the `createNodeField` is called. This should be called for all nodes (including those with missing images)